docs/source/_templates/sphinx_conf.template.py

```diff
@@ -34,12 +34,7 @@
 extensions = [
     "sphinx_design",
     "sphinx.ext.autodoc",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.napoleon",
-    "sphinx.ext.viewcode",
-    "sphinxarg.ext",
     "sphinxcontrib.autoprogram",
-    "m2r2",
```

Comment:
This package previously had an issue with dependency versions, which is why it was explicitly included. has that been resolved with all builds?